src/components/NavigationAccount.vue

```diff
@@ -140,6 +149,19 @@ export default {
 		iconError() {
 			return this.account.error ? 'icon-error' : undefined
 		},
+		quotaText() {
+			if (this.quota === undefined) {
+				return t('mail', 'Loading …')
+			}
+			if (this.quota === false) {
+				return t('mail', 'Not supported by the server')
```

Comment:
I would prefer not to show anything if quota is not supported or not available for any reason

Reply:
Our reasoning would be that this makes it more clear. Like if you were looking for this info as a user and the interface just hides it, you might think that it's hidden somewhere else, while you actually would have discovered it already.

In the end both is fine.

Reply:
If you have strong feeling about this please do not hesitate to open a ticket about it :)

Reply:
No, not at all ;-)